laravel/app/ClientCompany.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class ClientCompany extends Model
+{
+    protected $table = 'client_companies';
+    protected $primaryKey = 'id';
+    public $timestamps = false;
+
+    public function Items()
+    {
+    	return $this->belognsToMany('Item::class', 'items', 'end_user_id', 'id')
```

Comment:
綴りのミスが目立ってしまっているので、今一度確認するようにしましょう。
```suggestion
    	return $this->belongsToMany('Item::class', 'items', 'end_user_id', 'id')
```